common/lib/driver-definitions/src/storage.ts

```diff
@@ -302,7 +302,15 @@ export interface IDocumentServiceFactory {
     protocolName: string;
 
     /**
-     * Returns an instance of IDocumentService
+     * Creates the document service after extracting different endpoints URLs from a resolved URL.
+     *
+     * @param resolvedUrl - TODO
```

Comment:
Resolved Url contains the various info regarding the endpoints urls required to connect to those endpoints. It also contains info about the container/file on server like things required to locate it on server. For example, for odsp, it can contain siteUrl, driveId, itemId of the file on server.